src/test/javascript/display-deed.test.js

```diff
@@ -63,21 +63,11 @@ test('Test getLink()', async () => {
         link: LINK_NOT_NULL
     }));
 
+    delete window.location;
+    window.location = Object.create(window);
+    window.location.href = 'null';
```

Comment:
my js knowledge is non-existent, so I am curious - what are we doing here?

Reply:
JEST, the testing framework we use, gives off an error when you try to define a href link directly. By deleting the object, and creating it within the test, we can define and edit the properties internally.

Reply:
got it, thanks!